metamorph/metamorph_plugin.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import logging.config
+import json
+import time
+import os
+
+import requests
+
+
+class MetamorphPlugin(object):
+
+    MINUTE = 60
```

Comment:
I partly agree that it is clear but I don't like to have 'random' numbers in code. For example If you are a new Python programmer you would not know whether it is 60 seconds or 60 miliseconds. 
On the other hand in my case you do.  

Again when I read `time.sleep(self.MINUTE)` is telling me that it will wait one minute. Without 'any' brain activity. 

Your described case: 
1) We could use `SLEEPING_TIME` constant which can change. 
   + It is easy to maintain
   + No random numbers
   - Not that readable as `MINUTE`
2) Stay with `MINUTE`
  + Good readaility
  + No random numbers
   - In case of value change. Varaible name redefinition would be needed. (But still worth doing it to have readable and clean code.)
3) Have `60` value
  - Not that readable 
  - random number
  + Easy to maintain